packages/pharos/src/pages/item-detail/wc/item-carousel/item-carousel-item.ts

```diff
@@ -5,7 +5,7 @@ import type { Item } from '../../types';
 export const ItemCarouselItem = (item: Item): TemplateResult => html`
   <storybook-pharos-link href="#" subtle flex>
     <div class="item-detail-page__grid--carousel-item">
-      <img src="./images/item-detail/${item.image}" alt="${item.title}" />
+      <img src="${item.image}" alt="${item.title}" />
```

Comment:
A question for many of these URL changes: what let's us get rid of prefixes such as `./images/item-detail/`?

Reply:
This and I think one or two other files have a layer of indirection making it not obvious from the changes here; `item` comes from an object in another module, and each of those items' `image` was changed from a string to an imported image file.